src/sass/chrome.scss

```diff
@@ -106,40 +96,38 @@ aside {
 }
 
 // Header
-.pf-c-page__header {
-    overflow: visible;
-    .ins-c-dropdown-item__stack {
-        white-space: nowrap;
-        * { color: var(--pf-c-dropdown__menu-item--disabled--Color); }
-    }
-
-    .ins-c-dropdown-item__stack--header,
-    .ins-c-dropdown-item__stack--subValue {
-        font-size: var(--pf-global--FontSize--xs);
-    }
 
-    .ins-c-dropdown-item__stack--subValue { padding-top: var(--pf-global--spacer--xs); }
 
-    .pf-c-page__header-brand-link img { @include rem('height', 30px); }
-    .pf-c-page__header-brand-toggle svg {
-        @include rem('height', 21px);
-        @include rem('width', 21px);
-    }
-
-    .pf-c-page__header-tools .pf-c-avatar {
-        height: var(--pf-global--FontSize--4xl);
-    }
-}
-
-// Brand
-.pf-c-brand {
-    fill: #fff;
-    min-width: 200px;
-
-    @media screen and (min-width: $pf-global--breakpoint--md) {
-        width: 100%;
-        min-width: auto;
-    }
+.chr-c-masthead {
+  overflow: visible;
+  .chr-c-dropdown-item__stack {
+    white-space: nowrap;
+    * { color: var(--pf-c-dropdown__menu-item--disabled--Color); }
+  }
+  .chr-c-dropdown-item__stack--header,
+  .chr-c-dropdown-item__stack--subValue {
+    font-size: var(--pf-global--FontSize--xs);
+  }
+  .chr-c-dropdown-item__stack--subValue { padding-top: var(--pf-global--spacer--xs); }
+  .pf-c-page__header-brand-link img { @include rem('height', 30px); }
+  .pf-c-page__header-brand-toggle svg {
+    @include rem('height', 21px);
+    @include rem('width', 21px);
```

Comment:
It's a bummer none of these values match PF vars :disappointed: can we get some guidance of anyone from UX if we could use some values?

Reply:
@karelhala It's all obsolete anyway!